etc/concourse/scripts/cf-deploy-stage

```diff
@@ -58,13 +73,24 @@ echo "Mapping routes ..."
 mapRoutes abacus-usage-collector 6
 mapRoutes abacus-usage-reporting 6
 
-getApps
-
 if [ "$BIND_DB_SERVICE" == "true" ]; then
   echo "Binding services ..."
-  if [[ "$unamestr" == 'Linux' ]]; then
-    echo ${APPS[@]} | xargs -n1 | xargs -P 5 -i cf bind-service {} db
-  else
-    echo ${APPS[@]} | xargs -n1 | xargs -P 5 -n 1 -J {} cf bind-service {} db
-  fi
+  getApps abacus-usage-collector
```

Comment:
Listing apps can be a slow operation. Can we do the filtering in `bindService` instead?